### ISSUE TITLE
Code Migrations/Updates Welcome ?

### ISSUE THREAD

**Issue by oddlyspaced** [adolfintel/OpenPods#156]
First of all thanks a lot of creating this project. The app is super simple but serves a great purpose. I've seen other projects that make use of this as a base but I personally still use this for its simplicity.

Having checked out the code, the logic can be greatly improved and optimised at a lot of places. I'm personally willing to make updates to the codebase and migrate it bit by bit to kotlin if the original developers are okay with it.

**Comment by adolfintel** [comment]
Of course! Contributions are always welcome as long as you don't introduce things like dependencies on google services or analytics, etc.
